docs/compile.md

```diff
@@ -88,6 +88,8 @@ Alternatively, use [xcaddy](https://github.com/caddyserver/xcaddy) to compile Fr
 ```console
 CGO_ENABLED=1 \
 XCADDY_GO_BUILD_FLAGS="-ldflags='-w -s' -tags=nobadger,nomysql,nopgx" \
+CGO_CFLAGS=$(php-config --includes) \
+CGO_LDFLAGS="$(php-config --ldflags) $(php-config --libs)" \
```

Comment:
@dunglas Are these changes needed for building in Docker too?

https://github.com/dunglas/frankenphp/blob/a441e22a1bc06c49154a153d8137ec9050a03081/docs/docker.md?plain=1#L61-L70

Reply:
Btw, changing the `RUN` command to

```
RUN CGO_ENABLED=1 \
    XCADDY_GO_BUILD_FLAGS="-ldflags='-w -s' -tags=nobadger,nomysql,nopgx" \
    CGO_CFLAGS=$(php-config --includes) \
    CGO_LDFLAGS="$(php-config --ldflags) $(php-config --libs)" \
    xcaddy build \
…
```

doesn't seem to hurt here. I can make a PR for docs if it's actually needed.